backend/src/main/java/co/inventorsoft/scripty/controller/MockRequestController.java

```diff
@@ -0,0 +1,76 @@
+package co.inventorsoft.scripty.controller;
+
+import co.inventorsoft.scripty.model.dto.MockRequestDto;
+import co.inventorsoft.scripty.model.dto.StringResponse;
+import co.inventorsoft.scripty.service.MockRequestService;
+import com.fasterxml.jackson.core.JsonParseException;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.core.env.Environment;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.util.InvalidMimeTypeException;
+import org.springframework.web.bind.annotation.*;
+import javax.servlet.http.HttpServletRequest;
+import javax.validation.Valid;
+import java.nio.charset.Charset;
+import java.nio.charset.IllegalCharsetNameException;
+import java.util.regex.Pattern;
+
+/**
+ * @author A1lexen
+ */
+
+@Api(description = "Operations with mock requests")
+@RestController
+public class MockRequestController {
+    @Autowired
+    MockRequestService requestService;
+
+    @Autowired
+    Environment environment;
+
+    @ApiOperation(value = "Create mock response with specified parameters")
+    @PostMapping(value="/mock_request")
```

Comment:
Let's stick with following convention for url naming:
1) when we create something we use word in plural for url. E.g.: /mock-requests;
2) if you have multiple words split them with `-`.